File: 30 Days of code/src/fr/tenji/java/Days7.java
package fr.tenji.java;

import java.util.Scanner;

/**
 * Created by nekran on 11/10/16.
 */
public class Days7 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int[] arr = new int[n];
        StringBuilder s = new StringBuilder();
        StringBuilder s2 = new StringBuilder();
        for(int i=0; i < n; i++){
            arr[i] = in.nextInt();
        }
        in.close();
        for (int i = arr.length - 1; i >= 0 ; i--) {
            s.append(arr[i]);
            if (i != 0) {
                s.append(' ');
            }
        }
        System.out.println(s);
    }
}

File: 30 Days of code/src/fr/tenji/js/days7.js
/**
 * Created by nekran on 11/10/16.
 */
// process.stdin.resume();
process.stdin.setEncoding('ascii');

var input_stdin = '';
var input_stdin_array = '';
var input_currentline = 0;
var count = 1;
process.stdin.on('data', function (data) {
    input_stdin += data;
    count --;
    if (count <= 0) {
        process.stdin.end();
    }
});
process.stdin.on('close', function () {
    input_stdin_array = input_stdin.split('\n');
    main();
});

function readLine() {
    return input_stdin_array[input_currentline++];
}

/////////////// ignore above this line ////////////////////
function main() {
    // var n = parseInt(readLine());
    var arr = readLine().split(' ');
    console.log(arr);
    arr = arr.map(Number);
    var arr2 = [];
    for (var i = arr.length - 1 ; i >= 0; i--) {
        arr2.push(arr[i]);
    }
    console.log(arr2);
    console.log(arr2.join(' '));
}